operations/mimir-mixin/alerts/alertmanager.libsonnet

```diff
@@ -81,8 +81,8 @@
         {
           alert: $.alertName('AlertmanagerInitialSyncFailed'),
           expr: |||
-            increase(cortex_alertmanager_state_initial_sync_completed_total{outcome="failed"}[1m]) > 0
-          |||,
+            increase(cortex_alertmanager_state_initial_sync_completed_total{outcome="failed"}[%ss]) > 0
+          ||| % $.alertRangeInterval(1),
```

Comment:
can we move the `s` suffix of the string into `alertRangeInterval` or do we need a numeric output from `alertRangeInterval`?